docs/kpod-push.1.md

```diff
@@ -9,8 +9,9 @@ kpod push - Push an image from local storage to elsewhere
 **kpod** **push** [*options* [...]] **imageID** [**destination**]
 
 ## DESCRIPTION
-Pushes an image from local storage to a specified destination, decompressing
-and recompressing layers as needed.
+Pushes an image from local storage to a specified destination.
+Push is mainly used to push images to registries, however **kpod push** 
+can be used to save images to tarballs and directories using various transports.
```

Comment:
What transports?  You need to spell them out.

Reply:
fixed